jenkins/build_server_unix.sh

```diff
@@ -174,21 +177,25 @@ create_prop () {
 
 prep_artifacts () {
     # Prepare artifact directory for latestbuild publishing
-    pushd ${ARTIFACTS_DIR}
 
-    # Nexus strips "release" from ${PRODUCT}-${OS}-${VERSION}-release.${PKG_TYPE}.  It also transforms
-    # ${PRODUCT}-${OS}-${VERSION}-<debug|release>-symbols.${PKG_TYPE} to 
-    # ${PRODUCT}-${OS}-${VERSION}-symbols-<release|debug>.${PKG_TYPE}.  In order to minic this on
+    # Nexus strips "release" from ${PRODUCT}-${OS}-${SHA_VERSION}-release.${PKG_TYPE}.  It also transforms
+    # ${PRODUCT}-${OS}-${SHA_VERSION}-<debug|release>-symbols.${PKG_TYPE} to
+    # ${PRODUCT}-${OS}-${SHA_VERSION}-symbols-<release|debug>.${PKG_TYPE}.  In order to minic this on
     # latestbuild, these pkgs are renamed accordingly during copy
 
-    cp ${WORKSPACE}/${PRODUCT}-${OS}-${VERSION}-debug.${PKG_TYPE} ${PRODUCT}-${OS}-debug.${PKG_TYPE}
-    cp ${WORKSPACE}/${PRODUCT}-${OS}-${VERSION}-release.${PKG_TYPE} ${PRODUCT}-${OS}.${PKG_TYPE}
+    cp ${WORKSPACE}/${PRODUCT}-${OS}-${SHA_VERSION}-debug.${PKG_TYPE} ${ARTIFACTS_SHA_DIR}/${PRODUCT}-${OS}-debug.${PKG_TYPE}
+    cp ${WORKSPACE}/${PRODUCT}-${OS}-${SHA_VERSION}-release.${PKG_TYPE} ${ARTIFACTS_SHA_DIR}/${PRODUCT}-${OS}.${PKG_TYPE}
+
+    cp ${WORKSPACE}/${PRODUCT}-${OS}-${SHA_VERSION}-debug.${PKG_TYPE} ${ARTIFACTS_BUILD_DIR}/${PRODUCT}-${EDITION}-${VERSION}-${BLD_NUM}-${OS}-debug.${PKG_TYPE}
+    cp ${WORKSPACE}/${PRODUCT}-${OS}-${SHA_VERSION}-release.${PKG_TYPE} ${ARTIFACTS_BUILD_DIR}/${PRODUCT}-${EDITION}-${VERSION}-${BLD_NUM}-${OS}-release.${PKG_TYPE}
 
     if [[ ${OS} != 'ios' ]]; then
-        cp ${WORKSPACE}/${PRODUCT}-${OS}-${VERSION}-debug-symbols.${PKG_TYPE}  ${PRODUCT}-${OS}-symbols-debug.${PKG_TYPE}
-        cp ${WORKSPACE}/${PRODUCT}-${OS}-${VERSION}-release-symbols.${PKG_TYPE} ${PRODUCT}-${OS}-symbols-release.${PKG_TYPE}
+        cp ${WORKSPACE}/${PRODUCT}-${OS}-${SHA_VERSION}-debug-symbols.${PKG_TYPE}  ${ARTIFACTS_SHA_DIR}/${PRODUCT}-${OS}-symbols-debug.${PKG_TYPE}
+        cp ${WORKSPACE}/${PRODUCT}-${OS}-${SHA_VERSION}-release-symbols.${PKG_TYPE} ${ARTIFACTS_SHA_DIR}/${PRODUCT}-${OS}-symbols-release.${PKG_TYPE}
+
+        cp ${WORKSPACE}/${PRODUCT}-${OS}-${SHA_VERSION}-debug-symbols.${PKG_TYPE}  ${ARTIFACTS_BUILD_DIR}/${PRODUCT}-${EDITION}-${VERSION}-${BLD_NUM}-${OS}-symbols-debug.${PKG_TYPE}
+        cp ${WORKSPACE}/${PRODUCT}-${OS}-${SHA_VERSION}-release-symbols.${PKG_TYPE} ${ARTIFACTS_BUILD_DIR}/${PRODUCT}-${EDITION}-${VERSION}-${BLD_NUM}-${OS}-symbols-release.${PKG_TYPE}
```

Comment:
This should be removing "release" as well but it didn't before and it's not as important as the other location.

Reply:
removed release from changed ${PRODUCT}-${OS}-symbols-release.${PKG_TYPE} and ${PRODUCT}-${EDITION}-${VERSION}-${BLD_NUM}-${OS}-symbols-release.${PKG_TYPE}.